op-node/flags/flags.go

```diff
@@ -357,6 +357,29 @@ var (
 		Value:    time.Second * 1,
 		Category: SequencerCategory,
 	}
+	BuilderEnabledFlag = &cli.BoolFlag{
+		Name:     "l2.builder.enabled",
+		Usage:    "Enable the Builder API client",
+		Required: false,
+		EnvVars:  prefixEnvVars("L2_BUILDER_ENABLED"),
+		Value:    false,
+		Category: SequencerCategory,
```

Comment:
should it be in the Sequencer category? maybe it should be in a new builder category